packages/ui-color-picker/src/ColorMixer/ColorPalette/styles.ts

```diff
@@ -60,10 +60,10 @@ const generateStyle = (
       borderWidth: componentTheme.indicatorBorderWidth,
       borderColor: componentTheme.indicatorBorderColor,
       top: `${
-        state?.colorPosition?.y - px(`${props.indicatorRadius / 16}rem`)
+        state?.colorPosition?.y - 1 - px(`${props.indicatorRadius / 16}rem`)
```

Comment:
Because the indicator has a border so we need to subtract one more pixel to place the center of the indicator in the right position